systemtest/src/main/java/io/strimzi/systemtest/resources/operator/SetupClusterOperator.java

```diff
@@ -253,7 +264,19 @@ private void bundleInstallation() {
         // check if namespace is already created
         createCONamespaceIfNeeded();
         prepareEnvForOperator(extensionContext, namespaceInstallTo, bindingsNamespaces);
-        applyBindings();
+        // if a user want to customise one of the Role, ClusterRole, RoleBindings and ClusterRoleBinding he must do it
+        // everything by himself in scope of RBAC permissions otherwise we apply the default one
```

Comment:
This is not something a _user_ would do in system tests. Should it instead say _if they are managed directly by the individual test_?

Reply:
Yes, that's better :) I will update it.